source/Meadow-Filter-Parse.js

```diff
@@ -160,6 +160,33 @@ const addFilterStanzaToQuery = (pFilterStanza, pQuery) =>
 			}
 			break;
 
+    case 'FBJV':// Filter by JSON Value (left-side AND)
+      {
+      const jsonField = pFilterStanza.Field.split(',')[0];
+      const subFields = pFilterStanza.Field.split(',')[1];
+      pQuery
+        .addFilter('', '1', `LENGTH(${jsonField}) >`, 'AND', 'jsonFieldLen')
```

Comment:
This reads weird - why are these fields out of order? Is this a hack or workaround of some kind?

Reply:
it's a workaround. If I use `.addFilter('length(FormDataJson)', '1', '>', 'AND', 'FormDataJson_length')` the foxhound is adding Document.LENGTH(FormDataJson)